fhir-server-webapp/pom.xml

```diff
@@ -115,6 +115,22 @@
             <artifactId>jakarta.servlet-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-database-utils</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
+                <!-- exclude artifacts packaged in shared library dir -->
+                <exclusion>
+                    <groupId>com.ibm.db2</groupId>
+                    <artifactId>jcc</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.derby</groupId>
+                    <artifactId>derby</artifactId>
+                </exclusion>
```

Comment:
Maybe these should be marked provided at the top of the dependency list? 
That way they don't sneak in.

Reply:
will address this as part of https://github.com/IBM/FHIR/pull/1989